sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -2030,6 +2030,12 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
       Long.MinValue, Long.MaxValue, LongType.simpleString)(_.toLong)
   }
 
+  override def visitFloatLiteral(ctx: FloatLiteralContext): Literal = {
```

Comment:
Could you add a function description like the other functions around here, @yaooqinn ?

Reply:
done, thanks!